app/models/manageiq/providers/hawkular/middleware_manager.rb

```diff
@@ -203,18 +219,37 @@ def kill_middleware_domain_server(ems_ref)
       run_generic_operation(:Kill, ems_ref)
     end
 
-    def shutdown_middleware_server(ems_ref, params)
+    # server group ops
+    def start_middleware_server_group(ems_ref)
+      # blocking: boolean
```

Comment:
Blocking is optional param in the underlying Wildfly dmr api, but we never set it to true, so I removed the comment at all and didn't complicate the method signature with it.